pkg/app/load_opts_test.go

```diff
@@ -0,0 +1,24 @@
+package app
+
+import (
+	"reflect"
+	"testing"
+)
+
+// TestLoadOptsDeepCopy tests the DeepCopy function for LoadOpts struct.
+func TestLoadOptsDeepCopy(t *testing.T) {
+	lOld := LoadOpts{
+		Selectors:         []string{"test"},
+		RetainValuesFiles: true,
+		CalleePath:        "test",
+		Reverse:           true,
+		Filter:            true,
+	}
+	lNew := lOld.DeepCopy()
+	lNew.CalleePath = "test1"
```

Comment:
This seems non-sense because regardless of how DeepCopy is implemented, this will never modify `lOld` because neither lOld or lNew are pointers.